2-mandatory/4-people-i-know.js

```diff
@@ -394,6 +400,16 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+
+friendsWhoAreColleaguesOfStacie = friends
+  .filter((friend) => {
+    const findStacie = friend.colleagues.find(
+      (colleague) => colleague.name === "Stacie Villarreal"
+    );
+
+    return findStacie ? true : false;
+  })
+  .map((friend) => `${friend.name.first} ${friend.name.last}`);
```

Comment:
Good - there are few different ways of doing this, so if you have time, review the solution or your fellow trainees' solutions next week.